sw/device/lib/testing/i2c_testutils.c

```diff
@@ -31,10 +31,18 @@ void i2c_testutils_wr(dif_i2c_t *i2c, uint8_t addr, uint8_t byte_count,
   dif_i2c_fmt_flags_t flags = kDefaultFlags;
   uint8_t data_frame;
 
+  // The current function does not support initializing a write while another
+  // transaction is in progress
+
   // TODO: The current function does not support write payloads
   // larger than the fifo depth.
   CHECK(byte_count < I2C_PARAM_FIFO_DEPTH);
 
+  // TODO: "Callers should prefer `dif_i2c_write_byte()` instead, since that
+  //        function
+  //        provides clearer semantics. This function should only really be used
```

Comment:
I can clean up the line break.
This function uses dif_i2c_write_byte_raw() instead of dif_i2c_write_byte(). I'm quoting the dif's header here. I'll file an issue and point to it to make everything clearer.

Reply:
#15377 now has it as an issue, I'll add it to this TODO and resolve this when it shows as outdated.